app/models/import/variable_om_costs_per_full_load_hour_attribute.rb

```diff
@@ -0,0 +1,5 @@
+class Import
+  VariableOmCostsPerFullLoadHourAttribute =
+    Attribute.new('fixed_om_costs_per_year', 'variable_operation_and_maintenance_costs_per_full_load_hour')
```

Comment:
I think the `local_name` argument ("fixed_om_costs_per_year") may be wrong. Was it meant to be "om_costs_per_full_load_hour"?